src/api/helpers/update-applications-data/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Context, ObjectType } from '../../types';
+
+/**
+ * updateApplicationsData
+ * Updates many applications with the provided data
+ * @param {Context} context: KeystoneJS context API
+ * @param {String} applicationData: update applications data
+ * @returns {Promise<Object>} Created applications and reference number
```

Comment:
Should `applicationData` be `{Object}`?

Also looks like it's returning `Promise<Array<Application>>`?